olclient/cli.py

```diff
@@ -39,6 +39,8 @@ def argparser():
                         help='Configure ol client with credentials')
     parser.add_argument('--get-work', action='store_true',
                         help='Get a work by --title, --olid')
+    parser.add_argument('--get-author-works', action='store_true',
```

Comment:
Is this by author name or author olid? It looks like name maybe? We might want to make that explicit

Reply:
It works both ways,
if `--olid` option is specified, it shall take the Author's OLID `the` input.
if `--author` option is specified, it shall take the Author's name as the input.

Reply:
I shall update the help message to denote the same.

Reply:
Resolved changes requested in the review in commit: [8157a97](https://github.com/internetarchive/openlibrary-client/pull/136/commits/8157a97bab2e6b275fb1a2554ef891e1bc16debd)